### ISSUE TITLE
`slash-up sync` hangs after completion

### ISSUE THREAD

**Issue by linuswillner** [Snazzah/slash-up#189]
Whenever I run `npx slash-up sync`, the command will hang for an undefined amount of time despite apparently succeeding. I say undefined because I haven't tried leaving it running indefinitely to figure out whether it's actually going to ever exit, but anecdotal testing suggests it can stay hung for at least 15 minutes.

Placing a `console.log` at the end of the `handler()` function in `lib/commands/sync.js` seems to suggest that it reaches the end of the function no problem, so maybe there something is not terminating properly somewhere?

Here's the configuration I'm using (I logged out `process.env` to ensure the environment values referenced are defined).

```ts
module.exports = {
  token: process.env.DISCORD_BOT_TOKEN,
  applicationId: process.env.DISCORD_APP_ID,
  commandPath: './src/commands',
  env: {
    development: {
      globalToGuild: process.env.DEVELOPMENT_GUILD_ID
    }
  }
}
```

Since I run `npx slash-up sync` as a deploy step, this is quite annoying. I can of course manually patch over the issue with [timeout](https://man7.org/linux/man-pages/man1/timeout.1.html), but it'd still be nice to have a proper fix.

**Comment by Snazzah** [comment]
So it outputs as normal but fails to exit afterwards? Huh. I haven't ran into a problem like that, but does explicitly calling `process.exit` at the end fix the issue?

**Comment by linuswillner** [comment]
That's correct. I didn't try the process.exit() approach however. I could try though, I'll get back to you.

**Comment by linuswillner** [comment]
Sorry, I took way too long to get back to you on this issue, but yes, putting `process.exit()` at the end of the handler function does indeed exit the app. 🤔

**Comment by Snazzah** [comment]
Can you check to see if this is fixed on `v1.2.1`?

**Comment by linuswillner** [comment]
Yep, that's fixed it! Cheers!
